src/IntegratedExperts/BehatPhpServer/PhpServerContext.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+/**
+ * This file is part of the IntegratedExperts Behat tests.
+ *
+ * @file Context PhpServer for Behat.
+ */
+
+namespace IntegratedExperts\BehatPhpServer;
+
+use Behat\Behat\Context\Context;
+use Behat\MinkExtension\Context\MinkContext;
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Symfony\Component\Console\Exception\RuntimeException;
+
+/**
+ * Class PhpServerTrait.
+ */
+class PhpServerContext extends MinkContext implements Context
```

Comment:
please try removing `extends MinkContext` and see if test passes - there is noting here from Mink